pkg/api/testing/defaulting_test.go

```diff
@@ -225,45 +225,56 @@ func TestDefaulting(t *testing.T) {
 	sort.Sort(testTypes)
 
 	for _, gvk := range testTypes {
-		_, expectedChanged := typesWithDefaulting[gvk]
-		iter := 0
-		changedOnce := false
-		for {
-			if iter > *roundtrip.FuzzIters {
-				if !expectedChanged || changedOnce {
-					break
+		t.Run(gvk.String(), func(t *testing.T) {
```

Comment:
Yes.

It failed for ResourceClaimList, but I couldn't do a `dlv test ...  -- -test.run=...` where I debugged just that one failure.

Reply:
I have one more update for this: by creating the fuzzer anew for each sub-test, they become truly independent. The odd behavior from the commit message (failure when running all tests, passing when running just the sub-test) came from reusing the fuzzer because the randomness was different.